src/ucp/wireup/wireup_cm.c

```diff
@@ -239,6 +243,31 @@ ucp_cm_client_connect_prog_arg_free(ucp_cm_client_connect_progress_arg_t *arg)
     ucs_free(arg);
 }
 
+static uint64_t ucp_cm_client_restore_ep(ucp_wireup_ep_t *wireup_cm_ep,
+                                         ucp_ep_h ucp_ep)
+{
+    ucp_ep_h tmp_ep = wireup_cm_ep->tmp_ep;
+    ucp_wireup_ep_t *w_ep;
+    uint64_t tl_bitmap;
+    ucp_lane_index_t lane_idx;
+
+    for (lane_idx = 0; lane_idx < ucp_ep_num_lanes(tmp_ep); ++lane_idx) {
+        if (tmp_ep->uct_eps[lane_idx] != NULL) {
+            ucs_assert(ucp_ep->uct_eps[lane_idx] == NULL);
+            ucp_ep->uct_eps[lane_idx] = tmp_ep->uct_eps[lane_idx];
+            w_ep = ucs_derived_of(ucp_ep->uct_eps[lane_idx], ucp_wireup_ep_t);
+            w_ep->super.ucp_ep = ucp_ep;
+        }
+    }
+
+    tl_bitmap = ucp_ep_get_tl_bitmap(tmp_ep);
```

Comment:
better to call it separately since it's not related to "restore"

Reply:
ok, will try to update